pkg/eventmonitor/consumers/process.go

```diff
@@ -46,16 +46,28 @@ type event struct {
 var _ eventmonitor.EventConsumerHandler = &ProcessConsumer{}
 var _ eventmonitor.EventConsumer = &ProcessConsumer{}
 
+// ProcessConsumerOptions holds the options that can be passed to NewProcessConsumer
+type ProcessConsumerOptions struct {
+	// ID is the ID of the consumer that will be used to identify it with the event data stream
+	ID string
```

Comment:
_nit:_ actually I think this should be part of the ProcessConsumer and not in the ProcessConsumerOptions